server/src/termdb.server.init.js

```diff
@@ -61,6 +61,10 @@ export function server_init_db_queries(ds) {
 		'anno_categorical',
 		'buildDate'
 	]
+	const sampleColumns = listTableColumns(cn, 'sampleidmap')
+	if (!sampleColumns.includes('type'))
+		//Fix datasets with missing columns on the fly
+		cn.prepare('alter table sampleidmap add column type character varying(100)').run()
```

Comment:
this might break when pp has readonly access to the file, e.g. ppr for hpc-mount